pkgs/by-name/si/simpleBluez/package.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv
+, fetchFromGitHub
+, cmake
+, pkg-config
+, dbus
+, fmt_9
+, lib
+}:
+let
+  pname = "simpleBluez";
+  version = "0.7.3";
+  src = fetchFromGitHub {
+    owner = "OpenBluetoothToolbox";
+    repo = pname;
```

Comment:
Please don't use this.

`repo = pname;` is nice for DRY but creates a binding that goes too far.

See further information about this here: https://github.com/nix-community/nixpkgs-lint/issues/21